types/pg/pg-tests.ts

```diff
@@ -1,4 +1,5 @@
 import { types, Client, CustomTypesConfig, QueryArrayConfig, Pool } from 'pg';
+import TypeOverrides from 'pg/lib/type-overrides';
```

Comment:
I checked the source of `pg` again, and if I understand things correctly the reason we need this to be a separate file is because `TypeOverrides` is not exported from [`index.js`](https://github.com/brianc/node-postgres/blob/25f658f227a1bcbe759423678a7ab4ba8e067994/packages/pg/lib/index.js#L26-L32).

Would that be correct?

Reply:
Yes, this PR is introducing types that are not currently exported from `index.js`. Do you think that could be a problem?

Reply:
I'm not entirely sure... I think it's unorthodox, but I don't see a reason why it would create actual issues.
Perhaps it's a good idea to let a maintainer look at this, but I don't know of a way to do that other than to wait until the bot does it.

Reply:
@RyanCavanaugh Sorry to bother you for this, but your amazing bot seems to fall short here. Is there a way to "request human attention" that I'm unaware of?

Reply:
Do you mean reaching out to the pg module maintainers or asking a maintainer of this repo?

Reply:
The latter. I'm pretty sure that this describes the module accurately, but I'm not 100% sure that this is the "correct" or "orthodox" way to do things around here.